libvast/builtins/formats/zeek_tsv.cpp

```diff
@@ -702,7 +702,7 @@ auto parser_impl(generator<std::optional<std::string_view>> lines,
         .emit(ctrl.diagnostics());
     }
   }
-  if (document.builder->rows() > 0) {
+  if (document.builder and document.builder->rows() > 0) {
```

Comment:
This is a duplicate of a7268304b997cbb12735554c5e3e528b72af8929 in #3320.

Reply:
I know, I just added it here because the CI failed otherwise.